modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/rule-builder/FormsRuleList.es.js

```diff
@@ -304,45 +306,45 @@ const ListItem = ({dataProvider, fields, onDelete, onEdit, pages, rule}) => {
 						{Liferay.Language.get('if')}
 					</b>
 					{conditions.map((condition, index) => (
-						<>
+						<React.Fragment key={`fra${index}`}>
```

Comment:
Could you use `<>`?

Reply:
Hi @diegonvs, I'm getting warning on test console about key attr. Do you know other option to prevent this warnings ?

Reply:
Did you tried place a `key` on this component calling on parent component file? If you don't have a parent component you can wrap inside another component